shared/menubar/files-container.desktop.js

```diff
@@ -60,6 +42,22 @@ const mergeProps = (stateProps, dispatchProps) => ({
   }),
 })
 
+type TlfUpdateHocProps = {|
+  loadTlfUpdates: () => void,
+  userTlfUpdates: Array<UserTlfUpdateRowProps>,
+|}
+
+const TlfUpdateHoc = (ComposedComponent: React.ComponentType<any>) =>
+  class extends React.PureComponent<TlfUpdateHocProps> {
+    componentDidMount() {
+      this.props.loadTlfUpdates()
+    }
```

Comment:
If the content relies on explicit `loadTlfUpdates`, it seems we might also need to call it in `componentDidUpdate`.

Reply:
Hmm, that's called _all the time_. I'll have to make that part respond to notifications.

Reply:
Will do in a separate PR.